components/ActivityLogText.strings.json

```diff
@@ -12,5 +12,6 @@
   "PRESENCE_VERIFICATION_FAILED": "verification failed",
   "QRLOGIN_SUCCESFULL": "QRLogin successful",
   "WALLET_BINDING_SUCCESSFULL": "Activation successful",
-  "WALLET_BINDING_FAILURE": "Activation failed"
+  "WALLET_BINDING_FAILURE": "Activation failed",
+  "VC_REMOVED":"Removed from wallet"
```

Comment:
If I recall correctly, we've earlier(sometime before May) decided that we'd not commit or add the changes in `*.strings.json` file and instead only add the translation file changes in `locales/*.json`. If other members also clarify this, we can drop this change & maybe document this behaviour in a `CONTRIBUTING.md` file.

Reply:
*maybe as this is a bit redundant, we can take up the removal of the `*.strings.json` files sometime later in a separate card